audio/data.py

```diff
@@ -214,12 +232,19 @@ def _pre_process(self):
         
         if len(x.items) > 0:
             if not cfg.resample_to: _set_sr(x.items[0], x.config, x.path)
```

Comment:
This should technically be 
`if cfg.resample_to is None` but probably doesnt matter since we'll  never have 0 sample rate